resources/zib/ext-TreatmentDirective2.AdvanceDirective.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-TreatmentDirective2.AdvanceDirective" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-TreatmentDirective2.AdvanceDirective" />
+  <name value="ExtTreatmentDirective2AdvanceDirective" />
+  <title value="ext TreatmentDirective2.AdvanceDirective" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The AdvanceDirective concept of zib TreatmentDirective2." />
+  <purpose value="This extension represents the AdvanceDirective concept of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) TreatmentDirective2-v1.0(2020EN)](https://zibs.nl/wiki/TreatmentDirective2-v1.0(2020EN)). The cardinality of the `Consent.source[x]` element does not comply with the cardinality of the AdvanceDirective concept (NL-CM:2.2.8). This is solved by adding an extension that can be used to provide additional sources after the `Concent.source[x]` is populated first." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-treatmentdirective2-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/TreatmentDirective2-v1.0(2020EN)" />
+    <name value="zib TreatmentDirective2-v1.0(2020EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Consent" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension">
+      <path value="Extension" />
+      <comment value="The cardinality of the `Consent.source[x]` element does not comply with the cardinality of the AdvanceDirective concept (NL-CM:2.2.8) specified in the HCIM TreatmentDirective2. This is solved by adding an extension that can be used to provide additional sources after `Consent.source[x]` is populated." />
+      <min value="1" />

```

Comment:
Shouldn't the cardinality be 0..* instead of 1..* as this extension is only necessary if there are additional AdvanceDirectives? Currently Forge also gives a warning in ZibTreatmentDirective2 on Consent.extension:additionalAdvanceDirective due to the mismatch of minimum cardinality: 'The minimum cardinality is smaller than the resource's minimum cardinality (1). Cardinality may only be further restricted.'

Reply:
Fixed!